### PR TITLE
[stable/mongodb] Ability to add labels to service 

### LINKED ISSUE #20758: [stable/mongodb] Ability to add labels to service
**Is your feature request related to a problem? Please describe.**
 There is no way to add a custom label to the statefulset service 

**Describe the solution you'd like**
 Add helm value + template to allow custom labels

### PR DESCRIPTION
**Pull request by yindia**
#### Which issue this PR fixes
  - fixes #20758 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/mongodb/templates/pvc-standalone.yaml [hunk 1]
```diff
@@ -7,6 +7,9 @@ metadata:
     chart: {{ template "mongodb.chart" . }}
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
+    {{- with .Values.statefulsetLabels }}
+{{ toYaml . | indent 4 }}
```
Comment (javsalgar) [inline stable/mongodb/templates/pvc-standalone.yaml:11]: Wouldn't `{{- toYaml .Values.statefulsetLabels | nindent 4}}` achieve the same as the whole block?

**Review by javsalgar — COMMENTED** [review]
Thanks for the PR. Minor comment

File: stable/mongodb/templates/pvc-standalone.yaml (hunk 1 above)
Comment (yindia) [reply stable/mongodb/templates/pvc-standalone.yaml:11]: @javsalgar Hey, It's nice but it will failed the lint

Comment (yindia) [reply stable/mongodb/templates/pvc-standalone.yaml:11]: @javsalgar I created the mesh here that's why I close it and reopen it please check https://github.com/helm/charts/pull/20903.

**Pull request closed without merging**